.github/workflows/pipeline.yml

```diff
@@ -73,65 +69,35 @@ jobs:
         run: |
           bundle exec rake
 
+
+
   deploy:
     name: Deploy
+
     runs-on: ubuntu-latest
-    environment: 'Beta - Production'
+    environment: 'sandbox'
```

Comment:
Will the 'sandbox' part of this line (and line 9) ever change? If so would it be easier to extract it to some kind of variable so we only have to update it in one place?

Reply:
I'm not sure. It may well do when we introduce a production environment but that might also just be a different workflow. Think I'm happy to cross that bridge when we get to it.